db/migrate/20221206171338_change_ispf_theme_to_ispf_themes_array.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeIspfThemeToIspfThemesArray < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :activities, :ispf_theme, :ispf_themes
+    change_column :activities, :ispf_themes, :integer, array: true, using: "(array[ispf_themes]::integer[])"
```

Comment:
I've never had to do this before in Rails but I'm struggling to find documentation for how to use `using` here - did you find any?

This is more for my own understanding than anything else!

Reply:
This is where I found how to convert the data type while keeping the value: https://stackoverflow.com/questions/13879713/postgresql-alter-column-data-type-from-integer-to-integer-array